github-issues.go

```diff
@@ -195,6 +199,10 @@ func populateIssues(url string) {
 			eachGitIssue.Title = elem.Title
 			eachGitIssue.Repo = elem.RepositoryURL
 			eachGitIssue.Link = elem.HTMLURL
+			eachGitIssue.CreatedAt = elem.CreatedAt.Format(layout)
+			eachGitIssue.UpdatedAt = elem.UpdatedAt.Format(layout)
+			delta := elem.UpdatedAt.Sub(elem.CreatedAt)
+			eachGitIssue.Age = (int)(delta.Hours())
```

Comment:
I will address these one by one here : 

* s/GitPRs/GithubPRs  
* s/Repoissue/RepoIssue 
Ok i can do the renaming above. I accept that GitPRs could be called GithubPRs and RepoIssue is better than Repoissue.

>Also we need to have a valid reason to avoid using libraries such as this https://github.com/google/go-github because we are doing low level work here and should be avoided.

This option of using libraries first was explored. They are not consumable because they are not well supported for all the API access we will need. We even explored octokit, and several other libraries and in then end had to take this approach. Using the REST apis directly is ok in my opinion for buggy's needs. 

> var token = ""  --> This purpose is not clear
I will add a comment. All github requests need a token. 

> fmt.Println(token) --> Frivolous printing not needed, avoid debug statements in project repo.
Agreed. Will remove.

--> stray new line.  
Agreed. Will remove.

 auth() --> too generic name doesn't need to be a function IMHO, you can verify the token right here.  If we move to go-github all this problem also goes away. 

Actually i think its nice to have this in a separate function. I plan to add oauth option later. So its nice to have it modular. go-github does not give access to all the things i need in buggy.I will change the auth() to a better name.